expoDemo/MainScreen.js

```diff
@@ -1,59 +1,96 @@
-import React from 'react';
-import {ScrollView, TextInput} from 'react-native';
-import {Colors, View, Text, TouchableOpacity} from 'react-native-ui-lib';
+import React, {useCallback} from 'react';
+import {SectionList, StyleSheet} from 'react-native';
+import {Colors, View, Text, TouchableOpacity, Spacings, Image, Assets, Incubator} from 'react-native-ui-lib';
 import {menuStructure} from 'unicorn-demo-app';
 import _ from 'lodash';
 
+const {TextField} = Incubator;
+
+const sections = _.map(menuStructure, (section, key) => {
+  return {
+    key,
+    data: section.screens
+  };
+});
+
 export default function MainScreen({navigation}) {
   const [searchText, setSearchText] = React.useState('');
 
+  const renderSectionHeader = useCallback(({section}) => {
+    if (!_.find(section.data, screen => _.includes(screen.title.toLowerCase(), searchText.toLowerCase()))) {
```

Comment:
maybe extract the search logic `_includes(screen...` to avoid duplication in `renderItem`

Reply:
Fixed and improved search using fuzzy search